rust/integration-tests/src/sdk_tests/temp.rs

```diff
@@ -0,0 +1,140 @@
+#[allow(clippy::needless_return)]
+#[cfg(test)]
+mod tests {
+    use crate::{
+        diff_test_helper::{
+            processors::fungible_asset_processor::FungibleAssetProcessorTestHelper,
+        },
+        sdk_tests::{
+            setup_test_environment,
+            test_cli_flag_util::{parse_test_args, TestArgs},
+            validate_json,
+        },
+    };
+    use ahash::AHashMap;
+    use aptos_indexer_test_transactions::ALL_IMPORTED_TESTNET_TXNS;
+    use aptos_indexer_testing_framework::{
+        database::{TestDatabase},
+    };
+    use aptos_protos::transaction::v1::Transaction;
+    use once_cell::sync::Lazy;
+    use sdk_processor::{
+        config::{
+            db_config::{DbConfig, PostgresConfig},
+            indexer_processor_config::IndexerProcessorConfig,
+            processor_config::{
+                DefaultProcessorConfig, ProcessorConfig,
+            },
+        },
+        processors::fungible_asset_processor::FungibleAssetProcessor,
+    };
+    use std::{collections::HashSet};
+    use crate::sdk_tests::run_processor_test;
+    use crate::sdk_tests::get_test_config;
+    use crate::sdk_tests::DEFAULT_OUTPUT_FOLDER;
+    use std::collections::HashMap;
+
+    #[tokio::test]
+    async fn fa_processor_db_output_diff_test() {
```

Comment:
What's the difference between `fa_processor_db_output_diff_test` here and the one in `fungible_asset_processor`?

Reply:
this file should've included. removed